src/views/CreateType.vue

```diff
@@ -214,9 +215,17 @@ export default class Types extends Mixins(Translate) {
     this.currentlyEditingAttribute = -1;
   }
 
-  public createResourceType(): void {
-    ResourceTypes.create(this.newResourceType);
-    this.$router.push({ path: '/types' });
+  public async createResourceType(): Promise<void> {
+    try {
+      await ResourceTypes.create(this.newResourceType);
+      this.$router.push({ path: '/types' });
+    } catch (e) {
+      this.$notifications.create({
+        level: Level.Critical,
+        title: 'No connection to server: Resources type could not be created. Please try again.',
```

Comment:
Does it always has to be a lost connection?

Reply:
done